certified-connectors/AIForged/apiProperties.json

```diff
@@ -0,0 +1,66 @@
+{
+  "publisher": "Larc AI (PTY) Ltd",
+  "stackOwner": "Larc AI (PTY) Ltd",
+  "contact": {
```

Comment:
@aiforged The contact section is not required in the apiProperties.json file. Please go thru the instructions mentioned in the [documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#step-3-add-metadata).

Reply:
Step 3b states : Product or end service metadata and "contact" describes how users can contact the product or end service support resources for assistance or troubleshooting

Why are you questioning this if it optional and we provide this?